README.md

```diff
@@ -339,22 +339,30 @@ admin@ncs# devtools true
 
 Then do the configuration that you want to do through maagic API. Instead of commit use the `show configuration | display maagic` command. To continue with the authgroup example you would do:
 ```
+config
 devices authgroups group test default-map same-user same-pass
+top
 show configuration | display maagic 
 ```
 Output:
 ```
 admin@ncs(config)# devices authgroups group test default-map same-user same-pass
+admin@ncs(config)# top
 admin@ncs(config)# show configuration | display maagic 
 root = ncs.maagic.get_root(t)
 root.ncs__devices.authgroups.group.create('default')
-root.ncs__devices.authgroups.group['test'].default-map.create()
-root.ncs__devices.authgroups.group['test'].default-map.same-user.create()
-root.ncs__devices.authgroups.group['test'].default-map.same-pass.create()
+root.ncs__devices.authgroups.group['default'].default-map.create()
```

Comment:
I still don't get this output on the playground, not sure why. Could be because I'm using different groups ('test', 'default')?

```bash
>>> import ncs
>>> m = ncs.maapi.Maapi()  
>>> m.start_user_session('admin', 'system', []) 
>>> trans = m.start_write_trans()  
>>> root = ncs.maagic.get_root(trans)
>>> root.ncs__devices.authgroups.group.create('test')
ListElement name=group tag=1540685982 keys={test}
>>> root.ncs__devices.authgroups.group['test'].default_map.create()
Presence container name=default-map tag=1605796887 cached=False
>>> root.ncs__devices.authgroups.group['test'].default_map.same_user.create()
Empty leaf name=same-user tag=627058706 cached=False
>>> root.ncs__devices.authgroups.group['test'].default_map.same_pass.create()
Empty leaf name=same-pass tag=1062808443 cached=False
>>> trans.apply()
>>> 
developer:src > ncs_cli -C -u admin

admin connected from 127.0.0.1 using console on devpod-6753117695425925741-7d59bb5595-t4hw9
admin@ncs# show running-config devices authgroups 
devices authgroups group test
 default-map same-user
 default-map same-pass
!
admin@ncs# dev
Possible completions:
  devices    The managed devices and device communication settings
  devtools   Enable/disable development tools
admin@ncs# devtools true
admin@ncs# config
Entering configuration mode terminal
admin@ncs(config)# devices authgroups group test default-map same-user same-pass
admin@ncs(config-group-test)# top
admin@ncs(config)# show configuration | display maagic 
root = ncs.maagic.get_root(t)
admin@ncs(config)# 
```

Reply:
Fixed - using different group name in the second part now. If test is used both times NSO figures out that there is no need to create it again.